core/tests/puppeteer-acceptance-tests/utilities/user/logged-in-user.ts

```diff
@@ -109,10 +132,7 @@ export class LoggedInUser extends BaseUser {
    */
   async navigateToPreferencesPage(): Promise<void> {
     await this.goto(PreferencesPageUrl);
-  }
-
-  async navigateToPendingAccountDeletionPage(): Promise<void> {
-    await this.goto(PendingAccountDeletionPage);
+    await this.page.waitForTimeout(2147483647);
```

Comment:
Is this necessary? Or is it a function call you used for debugging only?

Reply:
Missed that, it was just for debugging. 
Removed.

Thanks.